### PR TITLE
Add Stackbit

### PR DESCRIPTION
**Pull request by JugglerX**
This pull request add's Stackbit to this theme. It add's the stackbit.yaml file which makes this theme work with several headless CMS automatically (including Forestry, NetlifyCMS, DatoCMS & Contentful) and it deploys the site to Netlify in 1 click.

You can try it out: https://app.stackbit.com/create?theme=https://github.com/stackbithq/moving (note the "create with stackbit" button in the readme will import the theme from your repo so until the PR is merged it wont work)

Hope you think it's as awesome as i do!

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -3,9 +3,9 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   {%- seo title=false -%}
-  <link rel="stylesheet" href="{{ site.url }}/assets/css/main.css" />
+  <link rel="stylesheet" href="/assets/css/main.css" />
   <link rel="shortcut icon" type="image/x-icon" href="{{ site.favicon }}" />
-  <link rel="stylesheet" href="{{ site.url }}/assets/css/agate.css" />
+  <link rel="stylesheet" href="/assets/css/agate.css" />
```
Comment (huangyz0918) [inline _includes/head.html]: Please keep the related path of your site, or it cannot find the .css files when deployed.

File: moving.gemspec [hunk 2]
```diff
@@ -18,6 +18,6 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "jekyll-feed", "~> 0.9"
   spec.add_runtime_dependency "jekyll-seo-tag", "~> 2.1"
 
-  spec.add_development_dependency "bundler", "~> 2.0.2"
+  spec.add_development_dependency "bundler", "2.0.1"
```
Comment (huangyz0918) [inline moving.gemspec:21]: Please keep bundled version ~> 2.0.2.

File: README.md [hunk 3]
```diff
@@ -105,6 +105,12 @@ Your theme is setup just like a normal Jekyll site! To test your theme, run `bun
 When your theme is released, only the files in `_layouts`, `_includes`, `_sass` and `assets` tracked with Git will be bundled.
 To add a custom directory to your theme-gem, please edit the regexp in `moving.gemspec` accordingly.
 
+## Stackbit Deploy
+
+This theme is ready to import into Stackbit in 1 click. This theme can be deployed to Netlify and you can connect any headless CMS including Forestry, NetlifyCMS, DatoCMS or Contentful.
```
Comment (huangyz0918) [inline README.md:110]: Change 1 click to: single click.

**Review by huangyz0918 — CHANGES_REQUESTED** [review]
Great feature @JugglerX, I have some tiny suggestions about your changes, please checkout and update. Thanks a lot.

**Comment by luhc228** [comment]
Please flx the ci error

**Comment by luhc228** [comment]
Please fix the conflict.

**Pull request closed without merging**